GCC/CORTEX_A53_64-bit_UltraScale_MPSoC/port.c

```diff
@@ -477,7 +477,8 @@ int32_t lReturn;
 /*-----------------------------------------------------------*/
 
 /* flag to control tick ISR handling, this is made true just before schedular start */
-uint64_t uxPortSchedularRunning = pdFALSE;
+
+volatile uint64_t uxPortSchedularRunning = pdFALSE;
```

Comment:
Used in main function:
int main(void)
{
    /* init SOC specific modules */
    System_init();
    Board_init();

    if (0 == portGET_CORE_ID())
    {
        /* This task is created at highest priority, it should create more tasks and then delete itself */
        gMainTask = xTaskCreateStatic( frertos_main,   /* Pointer to the function that implements the task. */
                                    "freertos_main", /* Text name for the task.  This is to facilitate debugging only. */
                                    MAIN_TASK_SIZE,  /* Stack depth in units of StackType_t typically uint32_t on 32b CPUs */
                                    NULL,            /* We are not using the task parameter. */
                                    MAIN_TASK_PRI,   /* task priority, 0 is lowest priority, configMAX_PRIORITIES-1 is highest */
                                    gMainTaskStack,  /* pointer to stack base */
                                    &gMainTaskObj ); /* pointer to statically allocated task object memory */
        configASSERT(gMainTask != NULL);

        vTaskCoreAffinitySet(gMainTask, 1);
        configASSERT(gMainTask != NULL);

        /* Start the scheduler to start the tasks executing. */
        vTaskStartScheduler();
    }
    else
    {
        while(ullPortSchedularRunning == 0)
        {
            ;
        }
        xPortStartScheduler();
    }

    /* The following line should never be reached because vTaskStartScheduler()
    will only return if there was not enough FreeRTOS heap memory available to
    create the Idle and (if configured) Timer tasks.  Heap management, and
    techniques for trapping heap exhaustion, are described in the book text. */
    DebugP_assertNoLog(0);

    return 0;
}

Reply:
Thanks! Considering this was a community supported port I figured this code would be in the community supported demos, looks like it was in the partner demos [here](https://github.com/FreeRTOS/FreeRTOS-Partner-Supported-Demos/blob/67fbd0020a5eb3dc917c04787b7d42fb9dd18d54/CORTEX_A53_64-BIT_TI_AM64_SMP/examples/test_demo/main.c#L44).